### ISSUE TITLE
Exporting as jar does not allow javafx to fully render

### ISSUE THREAD

**Issue by ajc-developer** [Jugal112/cs451_checkers#16]
![capture](https://cloud.githubusercontent.com/assets/8146286/17726277/0100991c-6420-11e6-865d-2f9414111510.PNG)

I read something online that it may be a bug with the version of javafx? Can't find the link anymore...

**Comment by dlaikhter** [comment]
We sorted this one out, right?

**Comment by ajc-developer** [comment]
90% sure we are exporting it correctly.  Something is up with javafx and exporting

**Comment by dlaikhter** [comment]
Ultimately this issue fell on my plate as the source was javascript related.

The issue was that changing the url with `window.location.href = 'url'` wouldn't work.
So That's the reason the screen would come up blank, but still load the css and run javascript.Therefore, it wouldn't load the templates.  This would have meant that I'd have to rework A LOT of the interface code.  It wasn't until after the millionth iteration, that I used angular's routing `$RouterProvider.otherwise( redirectTo: 'url' );`.  For some reason angular's url changing code worked, but it wouldn't work out of the box, so I used a library called ui-router which wraps up some of that stuff and has a method called `$state.go()` which lets you swap 'states' (templates).  

This is pretty much as specific as it gets for these kinds of bugs.
